dev-python/pycryptodome/pycryptodome-3.4.3.ebuild

```diff
@@ -0,0 +1,28 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+PYTHON_COMPAT=( python{2_7,3_4,3_5} )
+# needed to fix the: No such file or directory: 'build/test1.c'
+DISTUTILS_IN_SOURCE_BUILD="1"
+
+inherit distutils-r1 eutils
+
+DESCRIPTION="Self-contained Python package of low-level cryptographic primitives"
+HOMEPAGE="https://github.com/Legrandin/pycryptodome"
+SRC_URI="mirror://pypi/${PN:0:1}/${PN}/${P}.tar.gz"
+RESTRICT="test"
+
+LICENSE="BSD-2 Unlicense"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="test"
+
+RDEPEND="!dev-python/pycrypto"
```

Comment:
A bit more subtle approach is required here.
The number of packages that want pycryptodome is one: streamlink
The number of packages that want pycrypto is huge: https://qa-reports.gentoo.org/output/genrdeps/rindex/dev-python/pycrypto

Thus you are denying users from having streamlink and anything from the list above coinstalled.
One way to avoid this is to update all the packages from the list above to allow either pycrypto or pycryptodome.
But maybe there is a better way.